training/COMPARISON.md

```diff
@@ -0,0 +1,462 @@
+    'Age': ak.array([37, 35, 50, 32]),
+    'Salary': ak.array([75000, 77000, 100000, 35000])
+})
+ak_g = ak_df.GroupBy("F_Name")
+ak_g.count()
+(array(['John', 'Jane', 'Jake']), array([2 1 1]))
+```
+Notice the return here is the unique keys found in the column requested to group on along with the number of time each of those keys occured in the column. 
+
+<a id="ex"></a>
+## Example Application
+Now that we have highlighted some of Arkouda's key components, let's walk through an example of how to use them. A good example of this is walking through the code for a set operation on a sequence of `pdarray` objects. We will go through one of these functions, `setdiff1d`. The goal here is to identify the values only found in the first object provided.
+
+*NOTE: You will be using the `ak.concatenate()` function in these examples. This function creates a new array containing all values from the provided `pdarray`s.*
+
+For the example the following assumptions will be made:
+- The provided arrays are not unique
+- We will not be validating the format of the input as it is specifically designed for these examples
+- Input will be of the format below. Referenced as `m1` and `m2` in the examples.
+```python
+m1 =[
+    ak.array([0, 1, 3, 4, 8, 5, 0]),
+    ak.array([0, 9, 5, 1, 8, 5, 0])
+]
+m2 =[
+    ak.array([0, 1, 3, 4, 8, 7]),
+    ak.array([0, 2, 5, 9, 8, 5])
+]
+```
+
+First, import and connection to arkouda
+```python
+import arkouda as ak
+ak.connect()
+```
+
+Next, we will create a key to deinterleave the arrays. Using this to index into the concatenated arrays will return values from `m1`. We will also concatenate `m1` and `m2`. `ak.ones()` creates a `pdarray` with every value being `1`. `ak.zeros()` creates a `pdarray` with every value being `0`. 
+```python
+# Key for deinterleaving result
+isa = ak.concatenate(
+   (ak.ones(m1[0].size, dtype=ak.bool), ak.zeros(m2[0].size, dtype=ak.bool)), ordered=False
+)
+
+# concatenate m1 and m2
+c = [ak.concatenate(x, ordered=False) for x in zip(m1, m2)]
+```
+
+After this step, `isa` is a pdarray with `True` in the indexes where values are from `m1` and `False` in the indexes where values are from `m2`. This is because we cast the values from `1` and `0` to their equivalent boolean values using the `dtype=ak.bool` parameter.
+
+Now, we need to create a `GroupBy` object to get our unique keys. And the counts of those keys.
+```python
+g = ak.GroupBy(c)
+k, ct = g.count()
+```
+
+This next step does a lot for us, so we are going to get in more detail here. 
+```python
+truth = g.broadcast(ct == 1, permute=True)
+```
+We need to compute the unique keys that appear in the union only once, `ct == 1` provides that information. We then broadcast the resulting `True`/`False` value to the original keys the `GroupBy` was built from. Setting `permute=True` ensures that resulting boolean array is in the same order as the original keys, which is crucial to the next step.
```

Comment:
; instead of , here `only once; 'ct == 1'` as we're connecting two related independent clauses